### PR TITLE
Improved fraud function

### LINKED ISSUE #1727: Fraud Explanations[IMPROVE]
**What's the feature that should be improved?**
The fraud command is confusing.

**Describe how you would like the feature improved**
Explain what the figures mean.

**Possibly describe the ideal way to improve this**
Added some explanations.

**Additional information**
Add any other information o
Screen Shot 2022-04-20 at 9 54 35 PM
r screenshots about the feature improvement.

### PR DESCRIPTION
**Pull request by colin99d**
# Description

- [ ] Summary of the change / bug fix.
- Only shows sub calculations for MSCORE if they are requested (these are confusing)
- Adds color to show good, medium, and bad (makes numbers meaningful to user
- [ ] Link # issue, if applicable.
- Fixes #1727
- [ ] Screenshot of the feature or the bug before/after fix, if applicable.
Screen Shot 2022-05-22 at 2 12 24 PM

# How has this been tested?

Ran pytest

# Checklist:

- [x] Update [our Hugo documentation](https://openbb-finance.github.io/OpenBBTerminal/) following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/website).
- [x] Update our tests following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/tests).
- [x] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/CONTRIBUTING.md).
- [x] If a feature was added make sure to add it to the corresponding [scripts file](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/scripts).

# Others
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My code passes all the checks pylint, flake8, black, ... To speed up development you should run `pre-commit install`.
- [x] New and existing unit tests pass locally with my changes. You can test this locally using `pytest tests/...`.

### CODE REVIEW

File: openbb_terminal/stocks/fundamental_analysis/av_model.py [hunk 1]
```diff
@@ -403,14 +403,46 @@ def df_values(
...
+            row[i] = color_mscore(item)
+        elif name in ["Zscore", "McKee"]:
+            row[i] = color_zscore_mckee(item)
+        else:
+            row[i] = str(round(float(item), 2))
+
+    return row
+
+
+def color_mscore(value: str) -> str:
```
Comment (DidierRLopes) [inline openbb_terminal/stocks/fundamental_analysis/av_model.py:418]: docstring

File: openbb_terminal/stocks/fundamental_analysis/av_model.py [hunk 2]
```diff
@@ -403,14 +403,46 @@ def df_values(
     return values.tolist()
 
 
+def replace_df(name, row):
```
Comment (DidierRLopes) [inline openbb_terminal/stocks/fundamental_analysis/av_model.py:406]: docstring

File: openbb_terminal/stocks/fundamental_analysis/av_view.py [hunk 3]
```diff
@@ -226,18 +232,27 @@ def display_fraud(ticker: str, export: str = "", help_text: bool = False):
...
         Whether to show help text
+    color : bool
+        Whether to show color in the dataframe
+    detail : bool
+        Whether to show the details for the mscore
     """
-    df = av_model.get_fraud_ratios(ticker)
-    if df.empty:
+    try:
+        df, df_color = av_model.get_fraud_ratios(ticker, detail=detail)
```
Comment (DidierRLopes) [inline openbb_terminal/stocks/fundamental_analysis/av_view.py:241]: Should return just 1 dataframe.

And let's have color by default using rich

**Review by DidierRLopes — CHANGES_REQUESTED** [review]

File: openbb_terminal/stocks/fundamental_analysis/av_view.py [hunk 4]
```diff
@@ -251,6 +270,7 @@ def display_fraud(ticker: str, export: str = "", help_text: bool = False):
     if help_text:
         console.print(help_message)
     export_data(export, os.path.dirname(os.path.abspath(__file__)), "dupont", df)
+    return
```
Comment (DidierRLopes) [inline openbb_terminal/stocks/fundamental_analysis/av_view.py:273]: no need for this

File: openbb_terminal/stocks/fundamental_analysis/fa_controller.py [hunk 5]
```diff
@@ -781,11 +781,32 @@ def call_fraud(self, other_args: List[str]):
...
         )
+        parser.add_argument(
+            "-d",
+            "--detail",
+            action="store_true",
+            dest="detail",
+            default=False,
+            help="Shows the details for calculating the mscore",
+        )
+        parser.add_argument(
```
Comment (DidierRLopes) [inline openbb_terminal/stocks/fundamental_analysis/fa_controller.py:792]: let's have it true by default

File: openbb_terminal/stocks/fundamental_analysis/av_view.py [hunk 6]
```diff
@@ -216,7 +216,13 @@ def display_earnings(
 
 @log_start_end(log=logger)
 @check_api_key(["API_KEY_ALPHAVANTAGE"])
-def display_fraud(ticker: str, export: str = "", help_text: bool = False):
+def display_fraud(
+    ticker: str,
+    export: str = "",
+    help_text: bool = False,
+    color: bool = True,
```
Comment (DidierRLopes) [inline openbb_terminal/stocks/fundamental_analysis/av_view.py:223]: let's have color True and not being optional

File: openbb_terminal/stocks/fundamental_analysis/fa_controller.py (hunk 5 above)
Comment (colin99d) [reply openbb_terminal/stocks/fundamental_analysis/fa_controller.py:792]: Are you sure? These calculations are kind of "in the weeds" and will not be valuable to most people.

File: openbb_terminal/stocks/fundamental_analysis/av_view.py (hunk 4 above)
Comment (colin99d) [reply openbb_terminal/stocks/fundamental_analysis/av_view.py:273]: I was just doing it to match the other return so pylint would not throw an error.

File: openbb_terminal/stocks/fundamental_analysis/av_view.py (hunk 6 above)
Comment (colin99d) [reply openbb_terminal/stocks/fundamental_analysis/av_view.py:223]: Done!

**Review by DidierRLopes — APPROVED** [review]

**Pull request merged**